tox.ini

```diff
@@ -35,7 +35,7 @@ envlist =
     {py27,py34,py35,py36}-bottle{11,12}-webtest
     {py27,py34,py35,py36}-bottle-autopatch{11,12}-webtest
     {py27,py34,py35,py36}-cassandra{35,36,37,38}
-    {py27,py34,py35,py36}-celery{31,40}-redis{210}
+    {py27,py34,py35,py36}-celery{31,40,41,42}-redis{210}
```

Comment:
@alxch- that line was missing. Unfortunately we have to change the way we're running our test suite otherwise we have to remember to change many things to let tests run.